app/src/main/res/color/button_secondary_textcolor.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`textcolor` can be `text_color` in the file name, unless we're grouping the underscores by attributes moreover words?

Reply:
yes, my intention was grouping by attributes, but I can change if the other one is more convenient

Reply:
hmm, I gave it a second thought and i think it's better to group them by word, because in layout files, we also group by words. for instance: `activity_create_account`, NOT, `activity_createaccount`. I changed that for consistency. thanks for pointing this out.